kolibri/core/urls.py

```diff
@@ -4,12 +4,22 @@
 from __future__ import absolute_import, print_function, unicode_literals
 
 from django.conf.urls import url
+from kolibri.plugins import hooks
 
 from . import views
 
 app_name = 'kolibri'
 
 urlpatterns = [
-    url('^', views.IndexView.as_view(), name='index'),
+    url('^/$', views.IndexView.as_view(), name='index'),
     # url('.* ', views.TODOView.as_view())
 ]
+
+def retrieve_plugin_urls():
```

Comment:
Get all the URLs registered by plugins against the IMPORT_URLS hook - the functions registered should return the arguments for the django `url` function in a dict with `url_base` and `urls`.